src/main/java/org/alliancegenome/curation_api/model/entities/AffectedGenomicModel.java

```diff
@@ -80,9 +80,9 @@ public class AffectedGenomicModel extends GenomicEntity {
 	@IndexedEmbedded(includePaths = { "displayText", "formatText", "nameType.name", "synonymScope.name", "evidence.curie", "displayText_keyword", "formatText_keyword", "nameType.name_keyword", "synonymScope.name_keyword", "evidence.curie_keyword"})
 	@OneToOne(mappedBy = "singleAgm", cascade = CascadeType.ALL, orphanRemoval = true)
 	@JsonManagedReference
-	@JsonView({ View.FieldsOnly.class, View.AffectedGenomicModelView.class })
+	@JsonView({ View.FieldsOnly.class, View.AffectedGenomicModelView.class, View.ForPublic.class, View.ModelDocumentView.class })
```

Comment:
You don't need the ForPublic view on this... that view will be going away as "ForPublic" was only for the indexer and we have chosen have a view for each document Type.